src/getting-started.md

```diff
@@ -79,5 +107,182 @@ Target: 0000000dfff20000000000000000000000000000000000000000000000000000
 Eventually, the Translation Proxy log output will show sucessful share, which means you've run the configuration successfully!
 
 ```log
-2023-02-17T18:12:39.296514Z  INFO translator::proxy::bridge: SHARE MEETS DOWNSTREAM TARGET
+2023-03-13T11:59:43.396190Z  INFO translator_sv2::downstream_sv1::downstream: PROXY SERVER - ACCEPTING FROM DOWNSTREAM: 127.0.0.1:53260
+2023-03-13T11:59:43.396700Z  INFO translator_sv2::downstream_sv1::downstream: Down: Subscribing
+2023-03-13T11:59:43.397221Z  INFO translator_sv2::downstream_sv1::downstream: Down: Authorizing
+2023-03-13T11:59:49.118285Z  INFO translator_sv2::upstream_sv2::upstream: Up: Sending: Sv2Frame { header: Header { extension_type: 32768, msg_type: 27, msg_length: U24(41) }, payload: Some(Mining(SubmitSharesExtended(SubmitSharesExtended { channel_id: 2, sequence_number: 0, job_id: 0, nonce: 4031057692, ntime: 1678701333, version: 536870912, extranonce: Owned([0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]) }))), serialized: None }
+2023-03-13T11:59:49.119795Z  INFO translator_sv2::upstream_sv2::upstream: Up: Successfully Submitted Share
+
+```
+
+
+## Config 2: SV1 firmware > Translation Proxy JN (Job Negotiator) > SV2 Pool
+
+This configuration allows mining devices running SV1 firmware to connect to an SV2 Pool through a Translation Proxy (tProxy). In this case the proxy is designed also to implement the **Job Negotiator (JN)** features: it's able to negotiate the block templates to mine on with the **Pool-side JN**. In this configuration, **transaction selection** is done by the miners locally (through the tProxy JN) and then negotiated with the pool.
+In the following guide a Template Provider is installed locally on the same machine, to provide block templates to the JN.
+
+Job Negotiator follow the specification on this page: [https://github.com/stratum-mining/sv2-spec/blob/main/06-Job-Negotiation-Protocol.md](https://github.com/stratum-mining/sv2-spec/blob/main/06-Job-Negotiation-Protocol.md) 
+
+
+![Config2](/assets/config-d.svg)
+
+For the **MVP2 release** the high level description is as follows:
+
+1. **Proxy JN** connects with the **Pool JN**;
+2. **Proxy JN** sends ***AllocateMiningJobToken*** message to the **Pool JN**;
+3. **Pool JN** answers with a ***AllocateMiningJobToken.Success*** which contains an unique token to identify the job;
+4. **Proxy JN** connects with its chosen **Template Provider**; 
+5. **Template Provider** sends ***News Template*** and ***New PrevHash*** to the **Proxy**;
+6. **Proxy JN** sends ***CommitMiningJob*** message to the **Pool** which contains data of the Template;
+7. **Pool** answers ***CommitMiningJob.Success*** always [read the note];
+8. Proxy sends to its miners SetCustomMiningJob and let the miners start mining and sending shares to the pool
```

Comment:
```suggestion
```

Reply:
Removed it.
What do you think about the fact that in JN protocol specs linked in the guide (https://github.com/stratum-mining/sv2-spec/blob/main/06-Job-Negotiation-Protocol.md) there are messages which are not tackled in the MVP2 release?
Should I remove also the link to the specs from the Config D guide?